menuinst/win32.py

```diff
@@ -115,6 +115,20 @@ def quoted(s):
         return s
 
 
+def ensure_pad(name, pad="_"):
+    """
+
+    Examples:
+        >>> ensure_pad('conda')
+        '_conda_'
+
+    """
+    if not name or name[0] == name[-1] == pad:
+        return name
+    else:
+        return "%s%s%s" % (pad, name, pad)
```

Comment:
How will this behave when it sees a single pad at the start or end? It'll double that one up. Is that what you want?

Reply:
> Is that what you want?

I think that's what I wanted, yes.  Let me look again at this code now that I have the activate.bat hack 🤮 done.

Reply:
I don't think it matters in most cases.  Probably only matters in weird edge cases.  The old version of `ensure_pad()` (from conda, see https://github.com/conda/conda/pull/6817) did what you think it should.  When I was writing this, I got scared `ensure_pad()` was doing something it shouldn't.  I guess what you *want* for behavior can be ambiguous here.  But I'll go back to the original.